.circleci/config.yml

```diff
@@ -7,12 +7,16 @@ jobs:
   build:
     docker:
       # specify the version you desire here
-       - image: circleci/ruby:2.5.1-node-browsers
-      
-      # Specify service dependencies here if necessary
+      - image: circleci/ruby:2.5.1-node-browsers
+        environment:
+          PGUSER: root
+      # Specify service dependencies here if necessarydb/
       # CircleCI maintains a library of pre-built images
       # documented at https://circleci.com/docs/2.0/circleci-images/
-      # - image: circleci/postgres:9.4
+      - image: circleci/postgres:9.6.5-alpine-ram
```

Comment:
@heidar has suggested we use a more recent version of postgress, possibly 10.5 or 11.